api/v1/controllers/samples.js

```diff
@@ -302,7 +305,10 @@ module.exports = {
         .then((samples) => {
           samples.forEach((sample) => {
             if (!sample.isFailed) {
-              publisher.publishSample(sample, subHelper.model);
+              if (!featureToggles
+                .isFeatureEnabled(sampleStoreConstants.featureName)) {
+                publisher.publishSample(sample, subHelper.model);
+              }
             }
```

Comment:
what is the reason for doing it this way?
also, are we going to move the publishSample in delRlinksPromise to cache too?